src/Explorer/Tree/treeNodeUtil.ts

```diff
@@ -234,8 +227,8 @@ export const buildCollectionNode = (
     },
     onExpanded: async () => {
       // Rewritten version of expandCollapseCollection
-      useSelectedNode.getState().setSelectedNode(collection);
-      useCommandBar.getState().setContextButtons([]);
+      // useSelectedNode.getState().setSelectedNode(collection);
+      // useCommandBar.getState().setContextButtons([]);
```

Comment:
Intentional? Remove?

Reply:
Yep, this is intentional because it changes the target collection for New SQL Query/etc. when you expand a node even though the tree selection does not change. I want to do a quick second pass on that behavior before actually deleting it though. I'll do that and then delete these lines and push another iteration.

Reply:
Actually, in re-testing this behavior, I restored these lines. If we don't change the selected container on expand, the user has no way to change the selected container without opening a tab like Items or Settings.